extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/tasks/RememberTheMilkTasksImporter.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataportabilityproject.transfer.rememberthemilk.tasks;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import java.io.IOException;
+import java.util.UUID;
+import org.dataportabilityproject.spi.cloud.storage.JobStore;
+import org.dataportabilityproject.spi.transfer.provider.ImportResult;
+import org.dataportabilityproject.spi.transfer.provider.Importer;
+import org.dataportabilityproject.spi.transfer.types.TempTasksData;
+import org.dataportabilityproject.transfer.rememberthemilk.RememberTheMilkSignatureGenerator;
+import org.dataportabilityproject.transfer.rememberthemilk.model.tasks.ListInfo;
+import org.dataportabilityproject.transfer.rememberthemilk.model.tasks.TaskSeries;
+import org.dataportabilityproject.types.transfer.auth.AppCredentials;
+import org.dataportabilityproject.types.transfer.auth.AuthData;
+import org.dataportabilityproject.types.transfer.auth.TokenAuthData;
+import org.dataportabilityproject.types.transfer.models.tasks.TaskContainerResource;
+import org.dataportabilityproject.types.transfer.models.tasks.TaskListModel;
+import org.dataportabilityproject.types.transfer.models.tasks.TaskModel;
+
+public class RememberTheMilkTasksImporter implements Importer<AuthData, TaskContainerResource> {
```

Comment:
nit: can you add javadoc for the public classes?

Reply:
Done.